demo/client.cc

```diff
@@ -260,8 +313,42 @@ int main(int argc, char* argv[]) {
     return 1;
   };
 
+  ContentAnalysisAcknowledgement::FinalAction final_action =
+      ContentAnalysisAcknowledgement::ALLOW;
   for (int i = 0; i < datas.size(); ++i) {
-    HandleRequest(client.get(), i + 1, BuildRequest(datas[i]));
+    ContentAnalysisAcknowledgement::FinalAction final_action2;
+    HandleRequest(client.get(), i + 1, BuildRequest(datas[i]), &final_action2);
+    if (final_action2 > final_action)
+      final_action = final_action2;
+  }
+
+  if (group) {
+    std::cout << std::endl;
+    std::cout << "[Demo] Final action for all requests is ";
+    switch (final_action) {
+    case ContentAnalysisAcknowledgement::ACTION_UNSPECIFIED:
```

Comment:
Are we missing `std::cout << "unspecified";
                             break;`?

Reply:
Good question, but no!  Chrome's default action is fail open, so an unspecified action defaults to allow.  I added a comment to make this clearer.